content/news/releases/prestashop-8-1-5-maintenance-release.md

```diff
@@ -0,0 +1,98 @@
+twitter_image: /assets/images/2024/03/banner_8.1.5_1024x512.jpg
+icon: icon-leaf
+tags:
+- version
+- patch
+- releases
+- "8.1"
+- "8.1.x"
+---
+
+Another patch for PrestaShop 8.1 is available. This time without security updates but instead with some interesting improvements, as well as a number of bug fixes. In this article we'll lean into them because there's plenty to write about!
+
+![8.1.5 is available!](/assets/images/2024/03/banner_8.1.5_1534x424.jpg)
+
+PrestaShop 8.1.5 is available. This release fixes some bugs but also brings a lot of improvements. 
+
+It is recommended to upgrade your shop quickly in order to benefit from these fixes. Of course, don’t forget to backup before. Upgrade is possible using the latest version of the [1-Click Upgrade module](https://github.com/PrestaShop/autoupgrade/releases/).
+
+You can go to the [release page](https://github.com/PrestaShop/PrestaShop/releases/tag/8.1.5) to see a full changelog, but now let's dive into the details of this release.
+
+## Bug fixes
+
+Let's focus on some notable bug fixes that have been implemented in this version.
+
+### Improvements to the discounts marked as "highlight"
+
+Thanks to [@Hlavtox](https://github.com/Hlavtox), version 8.1.5 adds quite a few improvements to discounts added with the "Highlight" option. The previous implementation of this feature could cause unwanted discounts to appear in the available discounts in the user's account. The [pull request with a fix](https://github.com/PrestaShop/PrestaShop/pull/35464), clearly explains all the scenario and when, where, how discounts will display depending on this setting.
+
+### Fixes to deletion of customer's addresses
+
+Error "No carriers available". Does it ring a bell? This error was caused by a bug in the deletion of a customer's address. It has been fixed in [pull request #35017](https://github.com/PrestaShop/PrestaShop/pull/35017) from [@Hlavtox](https://github.com/Hlavtox).
+
+### Major bug fixes for the "Pack" products
+
+The "Pack" products feature has been improved in this version. There was a serious bug that caused the "Pack" products not to work corrently if they were set to "Decrement products in pack only". This has been fixed in [pull request #35372](https://github.com/PrestaShop/PrestaShop/pull/35372) from [@Codencode](https://github.com/Codencode) and [@Hlavtox](https://github.com/Hlavtox). After constructing a test scenario, they were able to identify the issue and fix it. Thank you to both of them for their hard work!
+
+### Better cart rules validation
+
+[@Hlavtox](https://github.com/Hlavtox) took the first step to improve the validation of cart rules. The [pull request #35166](https://github.com/PrestaShop/PrestaShop/pull/35166) introduced an extra check for cart rules that don't met the conditions. This will prevent the cart rules from being applied when they shouldn't be. This doesn't fix all the issues with cart rules, but it's a step in the right direction.
```

Comment:
```suggestion
[@Hlavtox](https://github.com/Hlavtox) took the first step to improve the validation of cart rules. The [pull request #35166](https://github.com/PrestaShop/PrestaShop/pull/35166) introduced an extra check for cart rules that don't meet the conditions. This will prevent the cart rules from being applied when they shouldn't be. This doesn't fix all the issues with cart rules, but it's a step in the right direction.
```